### PR TITLE
Reformat using @typescript-eslint/parser

### PR DESCRIPTION
**Pull request by pihart**

### CODE REVIEW

**Comment by cjquines** [comment]
what lint rule dislikes `props.`? i dont like it and i generally think that it should have `props.`

i also disagree that reassigning function arguments should be an error

File: src/lib/tools.ts [hunk 1]
```diff
@@ -151,12 +151,13 @@ export class LineHandler implements ToolHandler {
     y2: number,
     strict: boolean
   ): Promise<fabric.Line> => {
+    let [x, y] = [x2, y2];
     if (strict) {
       const rect = rectify(this.dirs, this.x, this.y, x2, y2);
-      x2 = rect[1];
-      y2 = rect[2];
+      x = rect[1];
```
Comment (cjquines) [inline src/lib/tools.ts:157]: i believe a lint rule wants us to turn this to `[, x, y] = rect;` and i like that lint rule where is that

Comment (pihart) [reply src/lib/tools.ts:157]: I like that too actually

**Comment by pihart** [comment]
I agree. It should not be an error and I have updated it. Not committing so that other changes can get incorporated.

File: src/lib/tools.ts (hunk 1 above)
Comment (pihart) [reply src/lib/tools.ts:157]: I think you're looking for https://eslint.org/docs/rules/prefer-destructuring

**Comment by cjquines** [comment]
i like destructuring yeet it in

**Pull request merged**